src/compas-ui/helpers/compas-scl-type-list.ts

```diff
@@ -0,0 +1,65 @@
+import {
```

Comment:
maybe move this to src/components

Reply:
we want to keep them in the helpers folder, they're solely being used by the UI components so that's why we wanna keep them in there